api-extractor.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  "mainEntryPointFilePath": "<projectFolder>/lib/index.d.ts",
+  "compiler": {
+  },
+  "apiReport": {
+    "enabled": false
+  },
+  "docModel": {
+    "enabled": false
+  },
+  "dtsRollup": {
+    "enabled": true,
+    "untrimmedFilePath": "<projectFolder>/dist/index.d.ts"
+  },
+  "tsdocMetadata": {
+    "enabled": false
+  },
+  "newlineKind": "lf",
+  "messages": {
+    "extractorMessageReporting": {
+      "ae-missing-release-tag": {

```

Comment:
probably it was bad idea, when I disabled that rule… instead of adding multiple exports - it should be better to just declare internal things as `@internal` in jsdoc

Reply:
We are not publishing the bundled version of types so this doesn't really matter?

Reply:
(chatted about this more, we can figure out later how to better use api-extractor, for now we just use it to bundle types)